opt4j-core/src/main/java/org/opt4j/core/config/visualization/Configurator.java

```diff
@@ -70,7 +70,7 @@ public void start(String[] args) throws ClassNotFoundException {
 		String filename = null;
 		Class<? extends Task> taskClass = null;
 		if (args.length < 1) {
-			throw new RuntimeException("Specify the task class");
+			throw new ClassNotFoundException("Specify the task class");

```

Comment:
ClassNotFound is used for classpath Issues. Does this fit here? We should check the spec of CNF. Alternatively, IllegalArgumentExcpetion or IllegalStateException could also fit.

Reply:
Since it checks for at least one task class to be given as an argument, I think IllegalArgumentException is the best fit. Corrected.